.github/workflows/sync_fork.yml

```diff
@@ -0,0 +1,22 @@
+name: Sync fork
+
+on:
+  schedule:
+    - cron:  '0 0 * * *' 
+
+  workflow_dispatch: 
+
+jobs:
+  sync_with_upstream:
+    runs-on: ubuntu-latest
+    name: Sync master with upstream latest
+
+    steps:
+    - name: Checkout tetrate
+      uses: actions/checkout@v2
+      with:
```

Comment:
I think we need to add fetch-depth: 0. Not sure if we would have all the branches and tags otherwise

Reply:
we dont actually fetch the tags here, but sure will check it out

Reply:
Gotcha. fetch tags takes care of it.